x/concentrated-liquidity/msg_server_test.go

```diff
@@ -184,13 +213,6 @@ func (suite *KeeperTestSuite) TestCollectFees_Events() {
 				suite.Require().ErrorContains(err, tc.expectedError.Error())
 				suite.Require().Nil(response)
 			}
-
-			// Some validate basic checks are defense in depth so they would normally not be possible to reach
-			// This check allows us to still test these cases
-			if tc.errorFromValidateBasic != nil {
-				suite.Require().Error(msg.ValidateBasic())
-				suite.Require().ErrorAs(msg.ValidateBasic(), &tc.errorFromValidateBasic)
-			}
```

Comment:
Why remove this? I think we should still test `msg.ValidateBasic()` somewhere. If this is required, please consider adding sufficient test coverage for `ValidateBasic`

Reply:
This code was added from someone else as a copy and paste and was never used in this test. The purpose of this check (in the other test this was copied from) was not to test ValidateBasic, but to jump over the ValidateBasic check to hit a branch that would otherwise be unreachable. 

With that being said, there is currently no ValidateBasic test for CollectFees or CollectIncentives (likely because the only check that exists for it right now is sender)